.github/workflows/build.yml

```diff
@@ -341,6 +346,11 @@ jobs:
         run: |
           npm run package  -- --target ${{ env.target }}
           cp *.vsix ../../.
+      - name: Package COBOL LS vsix for Web
+        working-directory: clients/cobol-lsp-vscode-extension
+        run: |
+          npm run package:web
+          cp *.vsix ../../.
```

Comment:
This will run 6x (once in each native configuration) and possibly overwrite the "desktop" VSIX package.
I suggest creating a separate job (e.g. buildWeb) in this file that will just do
- checkout
- "update version for PR" step
- npm ci
- npm run test:web
- npm run package:web
- upload artifact to preserve the VSIX

Reply:
It makes a separate vsix file with the name `cobol-language-support-web-[version]` where `version` is the latest numbered version. (2.1.2 as of this writing.)

Reply:
Made a separate job for building the web extension. Will need some help with the upload as there isn't an existing one to handle the artifact.

Reply:
https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/2c0b20fa249da679d33ac4fd9e5279dc1fd5eab4/.github/workflows/build.yml#L344-L348